src/commonMain/kotlin/exchange.dydx.abacus/output/Account.kt

```diff
@@ -2019,7 +2044,7 @@ data class TradingRewards(
                 Instant.fromEpochMilliseconds(lastStart.toLong())
             }
             while (obj.startedAtInMilliseconds < lastStartTime.toEpochMilliseconds().toDouble()) {
-                val previous = previousPlaceHolder(period, lastStartTime, obj.cumulativeAmount + obj.amount)
```

Comment:
this feels like a bug and when I made my tests it really seems like it shouldn't add amount here.

Reply:
These objects with zero amount shouldn't be adding to the older cumulative.